test/index.html

```diff
@@ -9,7 +9,33 @@
 <body>
   <div id="qunit"></div>
   <script src="../node_modules/qunitjs/qunit/qunit.js"></script>
-
+  <script type="text/javascript">
```

Comment:
This allows you to see the test results in Sauce Labs.
https://saucelabs.com/tests/68dcb9204fb94091996679fa2299f054

Should I just put this in globals-shim.js?